/*
** Copyright 2021 Bloomberg Finance L.P.
**
** Licensed under the Apache License, Version 2.0 (the "License");
** you may not use this file except in compliance with the License.
** You may obtain a copy of the License at
**
**     http://www.apache.org/licenses/LICENSE-2.0
**
** Unless required by applicable law or agreed to in writing, software
** distributed under the License is distributed on an "AS IS" BASIS,
** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
** See the License for the specific language governing permissions and
** limitations under the License.
*/

syntax = "proto3";

import "sasl.proto";

package Bloomberg.amqpprox.authproto;

message AuthResponse {
    enum AuthResult {
        ALLOW = 0;
        DENY = 1;
    }
    AuthResult result = 1;
    string reason = 2;
    SASL authData = 3;
}
